db/migrate/20180505152100_change_id_to_auto_increment_everywhere.rb

```diff
@@ -0,0 +1,33 @@
+class ChangeIdToAutoIncrementEverywhere < ActiveRecord::Migration
```

Comment:
Missing this migration from structure.sql.
If you edit it manually, please add this line to the end of file:
`INSERT INTO schema_migrations (version) VALUES ('20180505152100');`